src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int supply;
+    private int buy;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String line;
+        supply = 0;
+        buy = 0;
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+            line = reader.readLine();
+            while (line != null) {
+                String [] arrLine = line.split(",");
+                if (arrLine[0].equals("supply")) {
+                    supply += Integer.parseInt(arrLine[1]);
+                }
+                if (arrLine[0].equals("buy")) {
+                    buy += Integer.parseInt(arrLine[1]);
+                }
+                line = reader.readLine();
+            }
+        } catch (IOException e) {
+            System.out.println("File can't be reached");
+        } catch (NumberFormatException e) {
+            System.out.println("Wrong input data");
+        } catch (ArrayIndexOutOfBoundsException e) {
+            System.out.println("Wrong input data" + e);
```

Comment:
rethrow IOException as RuntimeException
you don't have to catch NumberFormatException and ArrayIndexOutOfBoundsException here